ui/documents/semantic-ui/templates/documents/form/FormFieldsContainer.jsx

```diff
@@ -185,19 +186,24 @@ const FormFieldsContainer = () => {
             id="NrDocs.Deposit.LicenseField.container"
             fieldPath="metadata.rights"
           >
-            <LocalVocabularySelectField
-              optimized
+            <LicenseField
+              searchConfig={{
+                searchApi: {
+                  axios: {
+                    headers: {
+                      Accept: "application/json",
```

Comment:
Why not use rdm (UI serialized) header? Is there something missing from ui serialization?

Reply:
@mirekys The problem is more or less the same as we have if everywhere with vocabulary items. If I use vnd headers, then I get back title as a string and this title is then stored in formik after I choose something, but when you wish to submit this to the server it fails, because server expects title to be an object. This is why Miro said we need to probably rethink how we handle serialization particularly in case of vocabularies.